aggregator/aggregator.go

```diff
@@ -3,28 +3,36 @@ package aggregator
 import (
 	"bytes"
 	"fmt"
+	"math"
 	"regexp"
 	"strconv"
 	"time"
 )
 
 type Func func(in []float64) float64
 
-func Sum(in []float64) float64 {
-	sum := float64(0)
-	for _, term := range in {
-		sum += term
-	}
-	return sum
-}
-
 func Avg(in []float64) float64 {
 	if len(in) == 0 {
 		panic("avg() called in aggregator with 0 terms")
 	}
 	return Sum(in) / float64(len(in))
 }
 
+func Delta(in []float64) float64 {
+	if len(in) == 0 {
+		panic("delta() called in aggregator with 0 terms")
+	}
+	return Max(in) - Min(in)
```

Comment:
Yeah, I hesitated and took the shortest path first. 
Anyway, re-worked that part now to have a single loop. Thanks.